Dockerfile

```diff
@@ -48,8 +49,6 @@ RUN apk add --no-cache \
 # Create the consumption directory
     mkdir -p $PAPERLESS_CONSUMPTION_DIR && \
 # Create user
-    addgroup -g 1000 paperless && \
```

Comment:
If you leave the comment here, you should move the command back down.

Reply:
Yes, it makes sense. Should be on the right place now.